js/src/collapse.js

```diff
@@ -132,7 +124,7 @@ const Collapse = (($) => {
       let activesData
 
       if (this._parent) {
-        actives = $.makeArray($(this._parent).find(this._selectorActives))
+        actives = $.makeArray($(this._parent).children().children(Selector.ACTIVES))
```

Comment:
The `find()` method in JQuery go through the whole tree of child (life child, grand-child, grand-grand-child).
The `children()` go through only one level. So 2 calls to `children()` will most likely be faster than `find()` in case there is content in the `.collapse` (that would be more than 2 level). Most likely there would be content in the `.collapse`.

In addition calling `children()` twice might actually be faster than `${childrenSelector} > .show, ${childrenSelector} > .collapsing` as it would loop on the child nodes in the tree rather than calling Sizzle/querySelector.